src/main/webapp/feedback.jsp

```diff
@@ -16,35 +16,77 @@
         <h1 class="text-center">Feedback</h1>
         <h4 class="text-center">Please submit your feedback for your interviewer below.</h4>
         <h5 class="text-center lead">Please enter a value between 1 and 10 (1 being strongly disagree and 10 being strongly agree).</h5>
-        <form name="feedbackForm" method="POST" onSubmit="addScheduledInterviewId()" action="/interviewer-feedback" class="needs-validation" novalidate>
+        <form name="feedbackForm" method="POST" onSubmit="addScheduledInterviewId()" action="/interviewer-feedback" class="needs-validation" onkeydown="return event.key != 'Enter';" novalidate>
           <div class="form-group">
             <div class="form-group form-inline"> 
               <label for="question1">I was comfortable during the interview:</label><br>
-              <input type="number" class="form-control" min="1" max="10" id="question1" name="question1" required><br>
+              <select class="form-control" name="question1">
+                  <option>Strongly Agree</option>
```

Comment:
Change the instructions to match this new format. By instructions l mean
"Please enter a value between 1 and 10 (1 being strongly disagree and 10 being strongly agree)."

Reply:
Is what I have now better?